ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -366,6 +367,16 @@ impl Mutation {
         Ok(true)
     }
 
+    async fn update_password(ctx: &Context, input: PasswordUpdateInput) -> Result<bool> {
```

Comment:
```suggestion
    async fn password_change(ctx: &Context, input: PasswordUpdateInput) -> Result<bool> {
```

Please also add unit test.